raiden/settings.py

```diff
@@ -39,7 +39,7 @@
 DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF = 1
 DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL_INITIAL = 5.0
 DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL_MAX = 60.0
-DEFAULT_TRANSPORT_MATRIX_SYNC_TIMEOUT = 20_000
+DEFAULT_TRANSPORT_MATRIX_SYNC_TIMEOUT = 60_000
```

Comment:
The value 20_000 was chosen because it was smaller than the presence timeout, with the assumption the timeout was compute from the beginning of subsequent sync calls. This, however, was an invalid assumption, while a /sync is pending on the matrix server the node does not go offline.

Here the best approach is to have the highest value possible, which will not have problems with firewalls and/or routers.